<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>information</class>
 <widget class="QDialog" name="information">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1566</width>
    <height>862</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="XDRGroup">
     <property name="title">
      <string>XDR</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="XDRTemp">
        <property name="text">
         <string>Temperature</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="XDRTempT">
        <property name="text">
         <string>Temperature théorique</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="XDRIndex">
        <property name="text">
         <string>Index</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="XDRPression">
        <property name="text">
         <string>Pression</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="XDRPitch">
        <property name="text">
         <string>Hauteur</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="XDRPRoll">
        <property name="text">
         <string>Profondeur</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="RotBox">
     <property name="title">
      <string>ROT</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="ROTRate">
        <property name="text">
         <string>Rate</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="ROTStatus">
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="MWVGroup">
     <property name="title">
      <string>MWV</string>
     </property>
     <widget class="QLabel" name="MWVWind">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>31</y>
        <width>491</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string>Angle du vent</string>
      </property>
     </widget>
     <widget class="QLabel" name="MWVReference">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>225</y>
        <width>491</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string>Reference</string>
      </property>
     </widget>
     <widget class="QLabel" name="MWVSpeed">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>450</y>
        <width>491</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string>Vitesse du vent</string>
      </property>
     </widget>
     <widget class="QLabel" name="MWVStatus">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>644</y>
        <width>491</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>540</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>noeuds</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>km/h</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>m/s</string>
       </property>
      </item>
     </widget>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QGroupBox" name="WIMDAGroup">
     <property name="minimumSize">
      <size>
       <width>523</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>MDA</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="WIMDAPression">
        <property name="text">
         <string>Pression (Inches)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="WIMDAPressionB">
        <property name="text">
         <string>Pression (bars)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="WIMDATemperature">
        <property name="text">
         <string>Température exterieur</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="WIMDAHumide">
        <property name="text">
         <string>Humidité relatif</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="WIMDAPDR">
        <property name="text">
         <string>Point de rosée</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="WIMDAVent">
        <property name="text">
         <string>Direction du vent</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="WIMDADirection">
        <property name="text">
         <string>Direction du vent magnétique</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="WIMDAVitesse">
        <property name="text">
         <string>Vitesse du vent</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="WIMDAVitesseVENT">
        <property name="text">
         <string>Vitesse du vent m/s</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
